lib/matplotlib/path.py

```diff
@@ -160,7 +164,8 @@ def __init__(self, vertices, codes=None, _interpolation_steps=1,
             self._readonly = False
 
     @classmethod
-    def _fast_from_codes_and_verts(cls, verts, codes, internals_from=None):
+    def _fast_from_codes_and_verts(cls, verts, codes, internals_from=None,
+                                   unmask_verts=True):
```

Comment:
This is a private function; do we even need the flag? Maybe a check for masked arrays is needed, but I'm pretty sure all internal callers already provide arrays.